packages/server/src/lib/email.js

```diff
@@ -159,7 +160,9 @@ async function sendGrantAssignedNotficationForAgency(assignee_agency, grantDetai
         title: 'Grants Assigned Notification',
         notifications_url: 'mailto:grants-helpdesk@usdigitalresponse.org?subject=Unsubscribe&body=Please unsubscribe me from the grant assigned notification email.',
     });
-    const emailPlain = emailHTML;
+
+    // TODO: add plain text version of the email
+    const emailPlain = emailHTML.replace(/<[^>]+>/g, '');
```

Comment:
This is a clever solution for the interim. I'd replace the matches here with a single space so that words/sentences that are separated by newlines aren't smooshed together.

Example:
```
Some information is here.
And this is also useful.
```
Should output as `Some information is here. And this is also useful.` rather than `Some information is here.And this is also useful.`